services/auth/src/signaturevalidator.js

```diff
@@ -88,7 +88,7 @@ const limitClientWithExt = function(credentialName, issuingClientId, accessToken
     }
 
     // Validate certificate scopes are subset of client
-    if (!utils.scopeMatch(scopes, [cert.scopes])) {
+    if (!utils.satisfiesExpression(scopes, {AllOf: cert.scopes})) {
```

Comment:
just a side note that I really like how much clearer the new syntax is to read

Reply:
Definitely, it was a good improvement..